### PR TITLE
fix: add support for wildcards for windows and pr testing workflow

### PR DESCRIPTION
**Pull request by derberg**
Hello Sir,

Few facts:
- I love the simplicity of your package
- I created this PR without first discussion under issue as I had the change done already for testing purposes (kinda to make sure I think this package can solve my problem). I'm ok if you do not want to accept this PR
- Package not working on windows if I use it as CLI in NPM script because of stup** windows that don't support wildcards
- I added glob to handle all paths with glob package and get support for wildcards on windows too
- I did not refactor code and there is for loop inside a for loop. I didn't do this because I'm lazy, I just think the code readability did not change and it is too trivial to bother with some refactoring. This is my personal opinion though and I'd love to make it nicer if you want me to
- change successfully tested in this PR https://github.com/asyncapi/asyncapi/pull/495

Well, that's it, have a good evening 😆 
Let me know man if I should do something more here 🍺

### CODE REVIEW

**Comment by derberg** [comment]
simplest pr testing workflow for GitHub Actions that you could use, includes testing on windows and Linux and mac

```
name: PR testing

on:
  pull_request:
    types: [opened, reopened, synchronize, ready_for_review]

jobs:
  test:
    if: github.event.pull_request.draft == false
    name: ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest, windows-latest]
    steps:
      - name: Checkout repository
        uses: actions/checkout@v2
      - name: Setup Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 14
      - name: Install dependencies
        run: npm install
      - name: Run test
        run: npm test
      - name: Run linter
        run: npm run lint
```

**Comment by derberg** [comment]
- removed travis
- added GH Actions workflow to test and lint on prs on different platforms and node versions
- had to do small adjustment in eslint rule because of common unix vs windows fight `Expected linebreaks to be 'LF' but found 'CRLF'`

I pushed the GH Action workflow from this PR to master of my fork and also created a PR there to see the workflow in action and make sure the current project passes everywhere https://github.com/derberg/mdv/pull/1 (don't worry about duplicates, it is because one workflow is in master and another in the PR)

File: .eslintrc.json [hunk 1]
```diff
@@ -79,10 +79,7 @@
         "key-spacing": "off",
         "keyword-spacing": "off",
         "line-comment-position": "off",
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
+        "linebreak-style": 0,
```
Comment (MikeRalphson) [inline .eslintrc.json:82]: Hmm - is there a reason for turning off this check?

Comment (derberg) [reply .eslintrc.json:82]: yeap, I wrote in general comment to the PR that without turning it off, linting on windows will always fail with `Expected linebreaks to be 'LF' but found 'CRLF'`

Comment (derberg) [reply .eslintrc.json:82]: https://stackoverflow.com/questions/37826449/expected-linebreaks-to-be-lf-but-found-crlf-linebreak-style

Comment (MikeRalphson) [reply .eslintrc.json:82]: Do you know if to use this workaround (https://stackoverflow.com/a/65519055) the whole .eslintrc.json file needs converting to .eslintrc.js ?

Comment (MikeRalphson) [reply .eslintrc.json:82]: Ah, it looks like you can do a `.eslintrc.js` file like:

```js
module.exports = {
    "extends": "./.eslintrc.json",
    "rules": {
        // override default options
       'linebreak-style': ['error', process.platform === 'win32' ? 'windows' : 'unix']
    }
};
```

Comment (MikeRalphson) [reply .eslintrc.json]: Looks like we're nearly there! Thanks for continuing to work on this.

Comment (derberg) [reply .eslintrc.json]: @MikeRalphson I pushed this new js file and looks good here https://github.com/derberg/mdv/pull/1 - all checks passed

**Review by MikeRalphson — APPROVED** [review]
LGTM! Thanks again

**Comment by MikeRalphson** [comment]
Released as `mdv@1.3.1`

**Pull request merged**